Podfile.lock

```diff
@@ -10,6 +10,7 @@ PODS:
     - ObjcExceptionBridging/ObjcExceptionBridging (= 1.0.1)
   - ObjcExceptionBridging/ObjcExceptionBridging (1.0.1)
   - PanModal (1.2.4)
+  - ReachabilitySwift (4.3.1)
```

Comment:
Alamofire also provides us [reachability checking] (https://github.com/Alamofire/Alamofire/blob/master/Source/NetworkReachabilityManager.swift). Consider to use it if we can, so we can reduce number of dependencies.

Reply:
Or [this](https://github.com/Daltron/NotificationBanner)